man/networkd.conf.xml

```diff
@@ -0,0 +1,123 @@
+    <productname>systemd</productname>
+
+    <authorgroup>
+      <author>
+        <contrib>Developer</contrib>
+        <firstname>Vinay</firstname>
+        <surname>Kulkarni</surname>
+        <email>kulkarniv@vmware.com</email>
+      </author>
+    </authorgroup>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>networkd.conf</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>networkd.conf</refname>
+    <refname>networkd.conf.d</refname>
+    <refpurpose>Global Network configuration files</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para><filename>/etc/systemd/networkd.conf</filename></para>
+    <para><filename>/etc/systemd/networkd.conf.d/*.conf</filename></para>
+    <para><filename>/usr/lib/systemd/networkd.conf.d/*.conf</filename></para>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>These configuration files control global network parameters.
+    For e.g. DHCP Unique Identifier (DUID).</para>
+
+  </refsect1>
+
+  <xi:include href="standard-conf.xml" xpointer="main-conf" />
+
+  <refsect1>
+    <title>[DUID] Section Options</title>
+
+    <para>This section configures the DHCP Unique Idendifier (DUID) value used by DHCP
+    protocol. DHCPv6 client protocol sends the DHCP Unique Identifier and the interface
+    Identity Association Identifier (IAID) to a DHCP server when acquiring a dynamic IPv6
+    address. DHCPv4 client protocol sends IAID and DUID to the DHCP server when acquiring
+    a dynamic IPv4 address if 'ClientIdentifier=duid'. IAID and DUID allows a DHCP server
+    to uniquely identify the machine and the interface making the DHCP IP request. To
+    configure IAID and ClientIdentifier, see <citerefentry><refentrytitle>systemd.network
+    </refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
+
+    <para>The DUID value specified here overrides the DUID that systemd-networkd
+    generates using the machine-id from the <filename>/etc/machine-id</filename> file.
+    To configure DUID per-network, see <citerefentry><refentrytitle>systemd.network
+    </refentrytitle><manvolnum>5</manvolnum></citerefentry>.</para>
+
+    <para>The configured DHCP DUID should conform to the specification in 
+    <ulink url="http://tools.ietf.org/html/rfc3315#section-9">RFC 3315</ulink>,
+    <ulink url="http://tools.ietf.org/html/rfc6355">RFC 6355</ulink>. To configure IAID, see
+    <citerefentry><refentrytitle>systemd.network</refentrytitle><manvolnum>5</manvolnum>
+    </citerefentry>.</para>
+
+    <para>The following options are available in <literal>[DUID]</literal> section:</para>
+
+    <variablelist class='network-directives'>
+
+      <varlistentry>
+        <term><varname>Type=</varname></term>
+        <listitem><para>The type of DUID specified in this section. The following values are
+        supported:</para>
+        <para>raw : If <literal>Type=raw</literal>, then <literal>RawData=</literal> specifies
```

Comment:
Yes. I'll add them to the documentation marking them as a future support item so that the intent of this configuration interface is not lost in a issues thread / exists in just my or Tom's head. I'll update IAIDValue references to just IAID and squash the commits within the next day or two.